### PR TITLE
UGENE-7619. Minor issue in warning if "Use pattern name" option during annotation creation is selected

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

File: src/corelibs/U2View/src/ov_sequence/find_pattern/FindPatternWidget.cpp [hunk 1]
```diff
@@ -678,15 +678,19 @@ void FindPatternWidget::updateErrorLabelState() {
                               messageFlagMap.contains(SequenceIsTooBig) ||
                               (messageFlagMap.size() == 1 && messageFlagMap.contains(PleaseInputAtLeastOneSearchPatternTip));
     if (hasNoPatternErrors) {
-        GUIUtils::setWidgetWarning(textPattern, false);
+        GUIUtils::setWidgetWarningStyle(textPattern, false);
     }
 }
 
 QString FindPatternWidget::buildErrorLabelHtml() const {
+    QString errorColor = Theme::errorColorLabelHtmlStr();
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_sequence/find_pattern/FindPatternWidget.cpp:686]: `static const QString ERROR_COLOR` and so on below

**Comment by DmitriiSukhomlinov** [comment]
GUI tests?

File: src/corelibs/U2View/src/ov_sequence/find_pattern/FindPatternWidget.cpp (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2View/src/ov_sequence/find_pattern/FindPatternWidget.cpp:686]: This is wrong. Theme is a dynamic object and should be re-read on use.

**Comment by yalgaer** [comment]
> GUI tests?

This case is already covered by test_3270. See the whole patch.

**Pull request merged**